docs/ansible/roles/debops.tinyproxy/getting-started.rst

```diff
@@ -0,0 +1,42 @@
+Getting started
+===============
+
+.. contents::
+   :local:
+
+
+Example inventory
+-----------------
+
+To install Tinyproxy on a host, it needs to be added to a specific Ansible
+inventory group:
+
+.. code-block:: none
+
+   [debops_service_tinyproxy]
+   hostname
+
+
+Example playbook
+----------------
+
+If you are using this role without DebOps, here's an example Ansible playbook
+that uses the ``debops.influxdb`` role:
```

Comment:
Don't forget to update the role name when you copy the documentation from something else. :-)